### PR TITLE
Configuring tensorboard log directory

### PR DESCRIPTION
**Pull request by navidmdn**
changed tensorboard argument of fit function from a boolean variable to tensorboard_dir as a string path
updated documents and tests

### CODE REVIEW

File: d3rlpy/algos/base.py [hunk 1]
```diff
@@ -228,8 +228,9 @@ def fit_online(
             logdir: root directory name to save logs.
             verbose: flag to show logged information on stdout.
             show_progress: flag to show progress bar for iterations.
-            tensorboard: flag to save logged information in tensorboard
-                (additional to the csv data)
+            tensorboard_dir: directory to save logged information in
+                tensorboard (additional to the csv data).  if None, the
```
Comment (takuseno) [inline d3rlpy/algos/base.py:232]: I'd like you to do close it with backslashes since this is visually better in the sphinx doc.
```
None -> ``None``
```

**Review by takuseno — CHANGES_REQUESTED** [review]
Thank you for your contribution! This is nice change! I left a minor comment. Please check.

**Review by takuseno — APPROVED** [review]
LGTM. Thanks for the change! I'll merge this after the test has passed.

**Comment by navidmdn** [comment]
> LGTM. Thanks for the change! I'll merge this after the test has passed.

Thanks for the fast review!

**Pull request merged**